stock_batch_picking/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# © 2012-2014 Alexandre Fayolle, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+    default_picker_id = fields.Many2one(
```

Comment:
Can we have this at the warehouse level instead? Having this at the company level will be an issue when multiple warehouse at separate locations come into play.